imm.yaml

```diff
@@ -59,23 +59,20 @@
     Id: da4ed776-0365-4d8a-a297-c4e91a14d646
     Measures:
         -
-            Score: 0
-            Description: None
-        -
+            Date: 2020-03-16
```

Comment:
Should this have a score?

Reply:
No, I modified the IMM badge code to calculate the score based on today's date so that the score drops automatically as time goes on.